### PR TITLE
Split custom elements into two features

### PR DESCRIPTION
**Pull request by foolip**
Fixes https://github.com/web-platform-dx/web-features/issues/1088.

### CODE REVIEW

File: features/autonomous-custom-elements.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Autonomous custom elements
+description: Autonomous custom elements are HTML elements with behavior that you define.
```
Comment (foolip) [inline features/autonomous-custom-elements.yml:2]: This description should say something about how to create these elements, tag names with a dash in them.

File: features/customized-built-in-elements.dist.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Generated from: customized-built-in-elements.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Customized built-in elements
+description: Customized built-in elements are HTML elements with behavior that you define.
```
Comment (foolip) [inline features/customized-built-in-elements.dist.yml:5]: This description should mention the `is` attribute.

File: features/customized-built-in-elements.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+name: Customized built-in elements
+description: Customized built-in elements are HTML elements with behavior that you define.
```
Comment (foolip) [inline features/customized-built-in-elements.yml:2]: This description should mention the `is` attribute.

File: features/autonomous-custom-elements.yml (hunk 1 above)
Comment (ddbeck) [reply features/autonomous-custom-elements.yml:2]: How about….

```suggestion
description: Autonomous custom elements are HTML elements with a hyphenated tag name (like `<example-element>`) that have a behavior you define.
```

File: features/customized-built-in-elements.yml (hunk 3 above)
Comment (ddbeck) [reply features/customized-built-in-elements.yml:2]: How about …

```suggestion
description: Customized built-in elements are HTML elements that extend built-in elements using the `is` attribute, to add new behaviors that you define.
```

**Review by ddbeck — COMMENTED** [review]
What do you think of a `custom-elements` group to put these together?

File: features/autonomous-custom-elements.yml (hunk 1 above)
Comment (foolip) [reply features/autonomous-custom-elements.yml:2]: Look good to me! An example tag name that's shorter and also realistic might be nice, but I struggle to find something that's obviously an example and not defining the scope of the feature. `<login-button>` and `<video-player>` aren't as good as `<example-element>`, so let's go with that.

File: features/customized-built-in-elements.yml (hunk 3 above)
Comment (foolip) [reply features/customized-built-in-elements.yml:2]: Thank you!

**Comment by foolip** [comment]
> What do you think of a `custom-elements` group to put these together?

I did consider that, but I hesitated because maybe we'll want a composite feature later to match the caniuse feature. But let me add the group and deal with that problem if it appears later. Some of the CSS pseudo-classes that aren't part of either feature now should probably go in the same group.

**Review by ddbeck — APPROVED** [review]
Thanks, Philip!

(For what it's worth, I also tried to find a more concrete example tag name but also found that it narrowed the apparent utility of the feature.)

**Comment by foolip** [comment]
The WPT side of this is done in https://github.com/web-platform-tests/wpt/pull/46309 and was quite hairy.

**Pull request merged**